packages/suite/src/support/extraDependencies.ts

```diff
@@ -28,5 +35,6 @@ export const extraDependencies: ExtraDependencies = {
                 state[backend.key].backends = backend.value;
             });
         },
+        storageLoadAccounts: (_, { payload }: StorageLoadAction) => payload.accounts,
```

Comment:
I am little bit worrying this will not work because if you do:

```
        builder.addCase(accountsActions.someAction, (state, action) => {
            return action.payload
        })
```

I think it will actually do nothing because `return action.payload` doesn't mean `state = action.payload`

Reply:
Please try to fix type in ExtraDependenciesType:
```
type StorageLoadReducer = (state: any, action: { type: any; payload: any }) => void;
// try to use never instead of void
type StorageLoadReducer = (state: any, action: { type: any; payload: any }) => never;
```
That should prevent this mistake to ever happen again.

Reply:
![Snímek obrazovky 2022-08-19 v 14 24 01](https://user-images.githubusercontent.com/2227592/185618633-cc8703a1-8ddd-43ec-a881-0fa6465a8d1b.png)